epub34/authoring/index.html

```diff
@@ -6541,93 +6482,38 @@ <h4>Structural semantics</h4>
+							<dt>Namespace:</dt>
 							<dd>
-								<dl class="elemdef" id="attrdef-epub-type-html">
-									<dt>Attribute Name:</dt>
-									<dd>
-										<p>
-											<dfn class="export" data-lt-no-plural="" data-dfn-type="element-attr">
-												<code>epub-type</code>
-											</dfn>
-										</p>
-									</dd>
+								<p>
+									<code>http://www.idpf.org/2007/ops</code>
+								</p>
+							</dd>
 
-									<dt>Usage:</dt>
-									<dd>
-										<p>Refer to the requirements for <a href="#sec-xhtml-structural-semantics"
-												>HTML</a>.</p>
-									</dd>
+							<dt>Usage:</dt>
+							<dd>
+								<p>Refer to the requirements for <a href="#sec-xhtml-structural-semantics">XHTML</a>, <a
+										href="#confreq-svg-structural-semantics">SVG</a>, and <a
+										href="#sec-overlays-def">media overlays</a>.</p>
+							</dd>
 
-									<dt>Value:</dt>
-									<dd>
-										<p>A [=ASCII whitespace|whitespace-separated=] [[infra]] list of values that
-											SHOULD be taken from the EPUB 3 Structural Semantics Vocabulary
-											[[epub-ssv-11]].</p>
-									</dd>
-								</dl>
+							<dt>Value:</dt>
+							<dd>
+								<p>A <a data-cite="xml#NT-S">whitespace-separated</a> [[xml]] list of [=compact URLs=],
+									with restrictions as defined in <a href="#sec-vocab-assoc"></a>.</p>
 							</dd>
 						</dl>
 
-						<div class="ednote">
-							<p>Support for an HTML serialization of the <code>epub:type</code> attribute depends on the
-								addition of support for the <a href="#sec-xhtml">HTML syntax</a> in the EPUB 3.4
-								revision. It is <b><i>at risk</i></b>, depending on authors' and implementers'
-								feedback.</p>
-
-							<p>The proposed <code>epub-type</code> will provide similar functionality for HTML content
-								documents but with some restrictions (namely, no extensibility through the
-									<code>epub:prefix</code> attribute). The attribute should be read as a replacement
-								for <code>epub:type</code> where this specification and the Reading Systems
-								specification [[epub-rs-34]] refer to that attribute for XHTML content documents. These
-								references will be updated once it is clearer that support will definitely be added in
-								this revision.</p>
-
-							<p>The <code>epub:type</code> attribute will remain the sole means of including semantics in
-								XML grammars in EPUB (i.e., XHTML content documents, SVG content documents, and media
-								overlay documents).</p>
-
-							<p>To provide feedback on this change, please open issues in the <a
-									href="https://github.com/w3c/epub-specs/issues">GitHub tracker for the EPUB
-									specifications</a>.</p>
-						</div>
-
-
 						<div class="caution">
 							<p>Although the <code>epub:type</code> attribute is similar in nature to the [^/role^]
 								attribute&#160;[[html]], the attributes serve different purposes. The values of the
```

Comment:
```suggestion
								attribute [[html]],
```
removing the entity for non-breaking space, since it is not used elsewhere

Reply:
Ya, the editor in me is torn on this. I know I never add them even though I feel bad for the poor orphans that result... :smile:

I just find it a nuisance to add the entities every time we have a reference, and I find it makes it harder to read with the respec shorthand on the reference, but do you feel strongly about having them @iherman? (I know you often add them.)

The looming problem if we try to standardize, of course, is that there's no consistency about whether the reference is more strongly tied to the text before or after it, so we can't just search and replace the problem away.

Reply:
I usually add them, but through Unicode and not with entities. I have a setup whereby an option+space (on my mac) puts it in the text. I do not know whether this is something you can set up in your environment...

And yes, the orphan references at the beginning of a line are a thorn in my eyes :-)

Reply:
No, I don't have anything like that. I have to insert the character from a menu or copy it in from elsewhere.

Now that I think of it, I was finding these as nbsp entities and then globalling them out periodically to 160 back when we had the xhtml ns declared. oxygen kept flagging them all as errors which made it harder to find real errors.

There's probably no real harm in adding a non-breaking space before all the references, as worst case it just pulls an extra word down. But we should do that in a separate pull request.

Reply:
There's some new CSS around preventing orphans, I wonder if that's a better solution than adding non-breaking spaces everywhere, but I imagine that's a respec change?

Reply:
I think this one's on us. I wasn't using orphan in the technical sense of the opening line of a paragraph being separated from the rest, just that the specification reference gets separated from what it refers to. For example:

```
                                   ... the `audio` element
[html]. ...
```

(Probably closer to a widow, I suppose, but still not something I think css can address.)

It may not be worth the bother. I skimmed probably a few hundred and only found a couple of spots where the reference ended up out of place.